### PR TITLE
[php] Add the `http_response_code()` function

### PR DESCRIPTION
**Pull request by cedx**

### CODE REVIEW

File: std/php/Global.hx [hunk 1]
```diff
@@ -1489,4 +1489,10 @@ extern class Global {
 		@see http://php.net/manual/en/function.cli-set-process-title.php
 	**/
 	static function cli_set_process_title(title:String):Bool;
+
+	/**
+		@see http://php.net/manual/en/function.http-response-code.php
+	**/
+	@:overload(function():Int {})
+	static function http_response_code(response_code:Int):Int;
```
Comment (RealyUniqueName) [inline std/php/Global.hx]: According to the doc return type should be `EitherType<Int,Bool>`

Comment (RealyUniqueName) [reply std/php/Global.hx]: And it's better to make the argument optional instead of adding `@:overload`

**Review by RealyUniqueName — CHANGES_REQUESTED** [review]

File: std/php/Global.hx (hunk 1 above)
Comment (CedX) [reply std/php/Global.hx]: Yes, I've already fixed it.

**Comment by cedx** [comment]
Thanks @RealyUniqueName

**Pull request merged**